spec/lib/ecco/row_event_listener_spec.rb

```diff
@@ -53,6 +53,9 @@
 
         context "UPDATE_ROWS" do
           let(:row_event_type) { EventType::UPDATE_ROWS }
+          let(:update_entry) { Java::JavaUtil::Map.entry([], []) }
```

Comment:
I can't get `Java::JavaUtil::Map.entry([], [])` to evaluate in Pry, is it used?

See:
```
[18] pry(main)> Java::JavaUtil::Map.entry([], [])
NoMethodError: undefined method `entry' for Java::JavaUtil::Map:Module
from (pry):18:in `evaluate_ruby'
```

Reply:
Works for me:

```shell
[1] pry(main)> Java::JavaUtil::Map.entry([], [])
=> #<Java::JavaUtil::KeyValueHolder:0x3e5d4f6b>
[2] pry(main)> 
```

Are you using pry with Jruby?

Reply:
> Works for me:

Interesting

> Are you using pry with Jruby?

Yep

```
[4] pry(main)> RUBY_PLATFORM
=> "java"
[5] pry(main)> Java::JavaUtil::Map.entry([], [])
NoMethodError: undefined method `entry' for Java::JavaUtil::Map:Module
from (pry):5:in `evaluate_ruby'
```

Did you just start `pry`? Hmm maybe I run some other JVM version

Reply:
```
$ java -version
java version "1.8.0_261"
Java(TM) SE Runtime Environment (build 1.8.0_261-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.261-b12, mixed mode)
```

Reply:
> Did you just start pry? Hmm maybe I run some other JVM version

Yep

```Shell
java -version
openjdk version "14.0.1" 2020-04-14
OpenJDK Runtime Environment (build 14.0.1+14)
OpenJDK 64-Bit Server VM (build 14.0.1+14, mixed mode, sharing)
```

Reply:
I tried with more recent JVM, and it works:

```
$ JAVA_HOME=/Users/robin/Library/Java/JavaVirtualMachines/openjdk-15.0.1/Contents/Home pry
[1] pry(main)> Java::JavaUtil::Map.entry([], [])
=> #<Java::JavaUtil::KeyValueHolder:0x5e95b0ab>
```

However, I believe we should only use things that works with Java 8 (as that's the oldest supported version of JVM for JRuby). We should likely change our travis config to run with version 8 too.

Reply:
Moreover, the real implementation uses an `AbstractMap.SimpleEntry`, see: 

https://github.com/osheroff/mysql-binlog-connector-java/blob/master/src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/UpdateRowsEventDataDeserializer.java#L70

It just just cast to a `Map.Entry`.

Reply:
Changed to correct entry here: 06c70093f58b461c0256d496e7378b4baa72ff4a06c70093f58b461c0256d496e7378b4baa72ff4a